modules/automatic-upgrade-failure.adoc

```diff
@@ -15,10 +15,10 @@ Some common reasons for the failure are:
 * The pre-flight checks may have failed, either because of insufficient RBAC permissions or because the cluster state is not recognizable.
```

Comment:
I wrote a doc-page for troubleshooting the upgrader - it would fit here perfectly, as this sentence covers only parts of possible reasons. Could we maybe link it here?

https://docs.openshift.com/acs/upgrading/upgrade-roxctl.html#troubleshooting-upgrader_upgrade-roxctl

Reply:
Does anyone know why this page is under "Configuring" and the other pages are under "Upgrading"? Would it make more sense for this to be moved under Upgrading with the rest of the upgrading type info? 
![Screenshot 2024-12-06 at 2 04 00 PM](https://github.com/user-attachments/assets/114d9e95-5fe6-46b1-95c3-50098473cbff)

Reply:
![Screenshot 2024-12-06 at 2 06 46 PM](https://github.com/user-attachments/assets/fcb1dc40-1022-4090-a44e-d42fa90fa1cd)

Reply:
It kind of makes sense to be placed there. The upgrader should be configured to run in the background and should do its job automatically when the trigger appears.

Upgrading sounds like a manual act of initiating the upgrade where human operator would conduct it.

I think that it may fit to both those categories if looked from the outside.